sync-core/src/test/java/com/cloudant/http/HttpTest.java

```diff
@@ -119,4 +122,52 @@ public void testReadBeforeExecute() throws IOException {
         Assert.assertEquals(bis.available(), data.getBytes().length);
         client.deleteDb();
     }
+
+
+    //NOTE: This test doesn't work with specified couch servers,
```

Comment:
Does this mean that this test case will sometimes fail depending on configuration? I'm confused as to when this should and shouldn't run